src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import Container from '../Container';
+import { HeaderStyled, BackButton, HeaderTitle } from './Header.styled';
+
+interface HeaderProps {
+  action?: boolean;
+  title: string;
+}
+
+const Header: FC<HeaderProps> = ({ action = false, title }) => {
+  return (
+    <Container>
+      <HeaderStyled>
+        {action && <BackButton to="/">Back</BackButton>}
+        <HeaderTitle>{title}</HeaderTitle>
+        <div>Name</div>
```

Comment:
I guess this div is no longer needed?

Reply:
The div is reserved for displaying email. I will also add the dropdown component here.